geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/LuceneRegionListener.java

```diff
@@ -107,7 +108,12 @@ public RegionAttributes beforeCreate(Region parent, String regionName, RegionAtt
   public void afterCreate(Region region) {
     if (region.getFullPath().equals(this.regionPath)
         && this.afterCreateInvoked.compareAndSet(false, true)) {
-      this.service.afterDataRegionCreated(this.luceneIndex);
+      try {
+        this.service.afterDataRegionCreated(this.luceneIndex);
+      } catch (LuceneIndexDestroyedException e) {
+        // @todo log a warning here?
```

Comment:
I guess we should log a warn, it should be super rare but it might be helpful when this does happen...

Reply:
+1